dashboard/app/controllers/activities_controller.rb

```diff
@@ -67,13 +67,16 @@ def milestone
         )
         if share_filtering_error
           FirehoseClient.instance.put_record(
-            study: 'share_filtering',
-            study_group: 'v0',
-            event: 'share_filtering_error',
-            data_string: "#{share_filtering_error.class.name}: #{share_filtering_error}",
-            data_json: {
-              level_source_id: @level_source.id
-            }.to_json
+            FirehoseClient::ANALYSIS_EVENTS_STREAM_NAME,
+            {
+              study: 'share_filtering',
+              study_group: 'v0',
+              event: 'share_filtering_error',
+              data_string: "#{share_filtering_error.class.name}: #{share_filtering_error}",
+              data_json: {
+                level_source_id: @level_source.id
+              }.to_json
+            }
```

Comment:
(optional) to reduce the diff (and/or to remove some curly braces) you could use Ruby's implicit conversion of keyword parameters into a hash passed as the last argument:

```ruby
FirehoseClient.instance.put_record(FirehoseClient::ANALYSIS_EVENTS_STREAM_NAME,
  study: 'share_filtering'
  [...etc...]
)
```